co-simulation/bundle/src/assembly/resources/scenarios/Town04/sumo/Town04.sumocfg

```diff
@@ -5,13 +5,12 @@
     </traci_server>
     <input>
         <net-file value="Town04.net.xml"/>
-        <route-files value="Town04.rou.xml"/>
-        <additional-files value="detector.xml"/>
-        <gui-settings-file value="gui.settings.xml"/>
```

Comment:
Why did we remove this. This file is required for setting the simulation speed for SUMO

Reply:
This was taken out by accident, added back in